.github/workflows/static_analysis.yml

```diff
@@ -21,6 +22,14 @@ jobs:
         with:
           python-version: '3.7'
           cache: 'pip' #Cache dependencies
+          cache-dependency-path: '**/pyproject.toml'
+
+      - name: Install pip-tools and generate requirements.txt
+        run: |
+          pip install pip-tools
+          #Generates a requirements.txt file from pyproject.toml 
```

Comment:
add a space here

Reply:
Where do you want the space, between 29 and 30?